prometheus-exporters/vrops-exporter/alerts/datastore.alerts

```diff
@@ -1,6 +1,19 @@
 groups:
 - name: datastore.alerts
   rules:
+  - alert: VVolDatastoreNotAccessible
+    expr: |
+      vrops_datastore_summary_datastore_accessible{type='vVOL'} == 0
+    for: 5m
+    labels:
+      severity: info
```

Comment:
I would replace critical with critical, unless there is a reason, why it shouldn't be critical anymore?